include/endian.h

```diff
@@ -29,12 +29,12 @@ static __inline uint16_t __bswap16(uint16_t __x)
 
 static __inline uint32_t __bswap32(uint32_t __x)
 {
-	return __x>>24 | __x>>8&0xff00 | __x<<8&0xff0000 | __x<<24;
+	return __x>>24 | (__x>>8)&0xff00 | (__x<<8)&0xff0000 | __x<<24;
 }
 
 static __inline uint64_t __bswap64(uint64_t __x)
 {
-	return __bswap32(__x)+0ULL<<32 | __bswap32(__x>>32);
+	return (__bswap32(__x)+0ULL)<<32 | __bswap32(__x>>32);
```

Comment:
I thing I'd prefer this:
```suggestion
	return uint64_t(__bswap32(__x))<<32 | __bswap32(__x>>32);
```
In c++23, we'll have [this](https://en.cppreference.com/w/cpp/numeric/byteswap) finally!

Reply:
`uint64_t` is not defined in that scope, I'll try to make it simple

Reply:
Are you sure?
<img width="553" alt="image" src="https://user-images.githubusercontent.com/1496907/236580539-e926d6ac-d33c-4f5c-9591-f9ba51455bb1.png">

Reply:
ooops, it was misleading error, thank you